src/Sylius/Behat/Context/Setup/GeographicalContext.php

```diff
@@ -38,25 +39,32 @@
     private $countryNameConverter;
 
     /**
+     * @var SharedStorageInterface
+     */
+    private $sharedStorage;
+
+    /**
      * @param FactoryInterface $countryFactory
      * @param RepositoryInterface $countryRepository
      * @param CountryNameConverterInterface $countryNameConverter
+     * @param SharedStorageInterface $sharedStorage
      */
     public function __construct(
         FactoryInterface $countryFactory,
         RepositoryInterface $countryRepository,
-        CountryNameConverterInterface $countryNameConverter
+        CountryNameConverterInterface $countryNameConverter,
+        SharedStorageInterface $sharedStorage
```

Comment:
Didn't we agree that `sharedStorage` should be first argument in the constructor?